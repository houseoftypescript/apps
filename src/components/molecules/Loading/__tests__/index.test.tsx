import { render } from '@testing-library/react';
import { Loading } from '../index';

describe('Loading', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Loading loading={true} error={null}>
        <></>
      </Loading>
    );
    expect(container).toMatchSnapshot();
  });
});
