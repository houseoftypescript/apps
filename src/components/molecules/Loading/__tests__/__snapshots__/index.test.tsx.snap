// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading to match snapshot 1`] = `
<div>
  <div
    class="border p-8 rounded"
  >
    <div
      class="flex items-center justify-center"
    >
      <span
        class="uppercase"
      >
        Loading
      </span>
    </div>
  </div>
</div>
`;
